test/Npgsql.Tests/ReaderNewSchemaTests.cs

```diff
@@ -83,17 +83,26 @@ public void BaseColumnNameWithColumnAliases()
                 );
             ");
 
-            var cmd = new NpgsqlCommand("SELECT Cod as CodAlias, Descr as DescrAlias, Date FROM data", conn);
+            var cmd = new NpgsqlCommand("SELECT Cod as CodAlias, Descr as DescrAlias, Date, NULL AS Generated FROM data", conn);
 
             using var dr = cmd.ExecuteReader(CommandBehavior.SchemaOnly | CommandBehavior.KeyInfo);
             var cols = dr.GetColumnSchema();
 
             Assert.That(cols[0].BaseColumnName, Is.EqualTo("cod"));
-            Assert.That(cols[0].ColumnName.ToString(), Is.EqualTo("codalias"));
-            Assert.That(cols[1].BaseColumnName.ToString(), Is.EqualTo("descr"));
-            Assert.That(cols[1].ColumnName.ToString(), Is.EqualTo("descralias"));
-            Assert.That(cols[2].BaseColumnName.ToString(), Is.EqualTo("date"));
-            Assert.That(cols[2].ColumnName.ToString(), Is.EqualTo("date"));
+            Assert.That(cols[0].ColumnName, Is.EqualTo("codalias"));
+            Assert.That(cols[0].IsAliased, Is.True);
+
+            Assert.That(cols[1].BaseColumnName, Is.EqualTo("descr"));
```

Comment:
Is this case testing anything not tested in the above case? If not we can remove.

Reply:
Looks like it duplicates other tests, but I do not want to touch this in that PR. We should prettify tests as a separate change with adding some new scenarios.